src/storage/src/store.rs

```diff
@@ -154,6 +191,20 @@ pub trait StateStore: Send + Sync + 'static + Clone {
         R: RangeBounds<B> + Send,
         B: AsRef<[u8]> + Send;
 
+    /// Opens and returns an iterator for given `prefix_key` and `suffix_range`.
```

Comment:
So maybe we should update the docs here? Could you please add some docs for other interfaces, mainly to show what `Option<_>` means for filter key?

Reply:
add some doc for `iter_inner` and `prefix_iter` at new comit